pkg/model/crd_test.go

```diff
@@ -1,9 +1,11 @@
-package model
+package model_test
 
 import (
 	"testing"
 
 	"github.com/stretchr/testify/assert"
+
+	"github.com/aws/aws-controllers-k8s/pkg/model"
 )
 
 func TestReplacePkgName(t *testing.T) {
```

Comment:
nit: we can move this unit test to a `types_test.go` file, since `ReplacePkgName` is defined in `types.go`

Reply:
yup. done.